curator-recipes/src/test/java/org/apache/curator/framework/recipes/locks/TestInterProcessReadWriteLock.java

```diff
@@ -239,6 +241,50 @@ public void testDowngrading() throws Exception
         }
     }
 
+    @Test
+    public void testContendingDowngrading() throws Exception
+    {
+        CuratorFramework client1 = CuratorFrameworkFactory.newClient(server.getConnectString(), new RetryOneTime(1));
+        CuratorFramework client2 = CuratorFrameworkFactory.newClient(server.getConnectString(), new RetryOneTime(1));
```

Comment:
nit: this test doesn't need two clients. You can refactor to use single client:

```java
        CuratorFramework client = CuratorFrameworkFactory.newClient(server.getConnectString(), new RetryOneTime(1));
```

Reply:
Fixup commit added.

Reply:
@kezhuw I see. I'm waiting for further review the last days.

@Randgalt comments on the Slack workspace that he may review this patch the next week. Let's wait for a while. If there's no objection I'll merge it later the next week.

Reply:
@kezhuw as you're added in the PMC, I'm wondering if you're willing to make the next release (5.4.1 or 5.5.0) following [the instructions](https://cwiki.apache.org/confluence/display/CURATOR/For+Curator+Committers#ForCuratorCommitters-ReleasingCurator).

Reply:
@tisonkun I am willing to take care of next release. It is also a chance for me to familiar with the release procedure. I will start next release cycle once this pr solved and PMC permission granted(which might take up to one week according to secretary's reply).